### DISCUSSION TITLE
Actores y casos de uso - Diagrama de contexto

### DISCUSSION THREAD

**Discussion by TheMoys** [TheMoys/24-25-IdSw1-SDR#24] [Q&A] [answered]
Creo esta discussion para hacer un brainstorming de los actores principales que atacan al sistema, asi como los principales casos de uso.

**Comment by diegojiimenez** [comment, marked as answer]
Actores que atacan el sistema:
- El Administrador general
- El responsable
- El destinatario
- Actor tiempo

Casos de uso para el administrador general:
- CRUD de usuarios
-  Descargar resumen de estados por los que paso el documento
- - Elegir que tipo de confirmación se espera del documento
- Adjuntar documento
- Crear/redactar mensaje
- Enviar mensaje
- CRUD de categorias

Casos de uso para el responsable:
- CRUD de destinatarios
-  Descargar resumen de estados por los que paso el doumento
- Elegir que tipo de confirmación se espera del documento
- Adjuntar documento
- Crear/redactar mensaje
- Enviar mensaje
- CRUD de categorias

Casos de uso para destinatarios:
- Abrir mensaje
- Cerrar visualización del documento
- Abrir Documento
- Firmar documento
- Confirmar lectura de documento
- Cerrar mensaje

Casos de uso actor tiempo:
- Enviar recordatorio de confirmacion de lectura al destinatario 
- Enviar recordatorio de firma al destinatario
